config/features.yml

```diff
@@ -1003,6 +1003,10 @@ features:
     actor_type: user
     description: Allows veterans to view their VA and Community Care appointments
     enable_in_development: true
+  va_online_scheduling_booking_exclusion:
+    actor_type: user
+    description: Permits the exclusion of Lovell sites from being scheduled prior to Oracle Health cutover
+    enable_in_development: true 
```

Comment:
There's an extra space at the end of this line. Can you remove please?

Reply:
Thanks for the review Eric, I've removed the extra space at the end.